rootfs/setup_rootfs.sh

```diff
@@ -55,7 +55,7 @@ echo
 # TODO: we really should not be using root for everything
 #       we should create a normal user and disable the root account, 
 #       or at least give it a strong password...
```

Comment:
Do we want to get rid of or update this TODO comment?

Reply:
Removed with commit 7f844df
Added new issue #28